Formula/jing.rb

```diff
@@ -1,7 +1,7 @@
 class Jing < Formula
   desc "RELAX NG validator"
-  homepage "https://code.google.com/p/jing-trang/"
-  url "https://jing-trang.googlecode.com/files/jing-20091111.zip"
+  homepage "https://code.google.com/archive/p/jing-trang/"   #"https://code.google.com/p/jing-trang/"
```

Comment:
If the URLs have changed, you don't need the comments; just the new values is fine.